src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsassessforearlyreleaseapi/service/policy/model/Criterion.kt

```diff
@@ -1,15 +1,26 @@
 package uk.gov.justice.digital.hmpps.hmppsassessforearlyreleaseapi.service.policy.model
 
+fun interface EvaluationStrategy {
+  fun isMet(answers: Map<String, Boolean>): Boolean
+
+  companion object {
+    val MET_IF_ALL_ARE_FALSE = EvaluationStrategy { answers -> answers.values.all { !it } }
```

Comment:
this is the default..
swapped the logic round too - it used to be eligible if all were answered true